subsys/bluetooth/audio/tbs.c

```diff
@@ -1990,8 +1990,7 @@ int bt_tbs_set_bearer_technology(uint8_t bearer_index, uint8_t new_technology)
 {
 	struct service_inst *inst = inst_lookup_index(bearer_index);
 
-	if (new_technology < BT_TBS_TECHNOLOGY_3G ||
-	    new_technology > BT_TBS_TECHNOLOGY_IP) {
+	if (new_technology < BT_TBS_TECHNOLOGY_3G || new_technology > BT_TBS_TECHNOLOGY_WCDMA) {
```

Comment:
Did this change get in by mistake?

Reply:
Unfortunately not. `BT_TBS_TECHNOLOGY_IP` is not a valid value for the technology field, and was removed. This unfortunately requires changes in the implementation as well. 

I can revert the removal of `BT_TBS_TECHNOLOGY_IP` to avoid this if wanted.

Reply:
Why not send it as a bugfix then so it can get in before freeze?

Reply:
Raised issue (https://github.com/zephyrproject-rtos/zephyr/issues/75646) and fixed by https://github.com/zephyrproject-rtos/zephyr/pull/75645

Of course this will now create a merge conflict, so let's hope that reviewers are active enough that this doesn't end up making this miss the release :)